OutOfSchool/OutOfSchool.WebApi/appsettings.Development.json

```diff
@@ -41,22 +41,23 @@
     "User": "admin",
     "Password": "admin"
   },
-  "CommonImagesRequestLimits": {
-    "MaxCountOfAttachedFiles": 30
-  },
   "Images": {
     "Workshop": {
-      "MinWidthPixels": 350,
-      "MaxWidthPixels": 10000,
-      "MinHeightPixels": 250,
-      "MaxHeightPixels": 8000,
-      "MaxSizeBytes": 16777216,
-      "MaxWidthHeightRatio": 2, 
-      "SupportedFormats": [
-        "jpeg",
-        "png"
-      ]
+      "Specs": {
```

Comment:
if the config is the same as in basic `appsettings.json` you don't really need to repeat this in other settings files.